modules/providers/service_container_annotation_discovery/tests/modules/service_container_annotation_discovery_test/src/Plugin/Plugin3/Plugin3A/Plugin3A.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Drupal\service_container_annotation_discovery_test\Plugin\Plugin3\Plugin3A;
+
+use Drupal\Component\Annotation\Plugin;
+use Drupal\Component\Plugin\PluginBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class Plugin3A
+ *
+ * @Plugin(
+ *   id = "Plugin3A",
+ *   label = "Label Plugin3A"
+ * )
+ *
+ * @package Drupal\service_container_annotation_discovery_test\Plugin\Plugin3\Plugin3A
+ */
+class Plugin3A extends PluginBase implements ContainerFactoryPluginInterface {
+  /**
+   * @inheritdoc
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return 'Hello World!';
```

Comment:
No, that is wrong.

This needs to be:

return new static($configuration, $plugin_id, $plugin_definition, 'Hello World');

then add a __construct here, that populates $this->data and add a getData() method.